product-backend/src/test/java/validator/RegisterValidatorTest.java

```diff
@@ -0,0 +1,107 @@
+package validator;
+
+import org.junit.jupiter.api.Test;
+import org.kainos.ea.cli.RequestEmployee;
+import org.kainos.ea.validator.RegisterValidator;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RegisterValidatorTest {
```

Comment:
You could try to use parametrised test
https://www.baeldung.com/parameterized-tests-junit-5